src/rabbit_lvc_plugin_m2k_converter.erl

```diff
@@ -0,0 +1,91 @@
+-module(rabbit_lvc_plugin_m2k_converter).
```

Comment:
This fine is missing license headers, plus we generally do not use `plugin` in module names, `rabbit_lvc_m2k_converter` should be unambiguous.